### one single ground target state calculated by position_calculator

Header header
int16 id                                # target id
float64 probability                     # target probability
string Class                            # target class name
sensor_msgs/NavSatFix global_position   # target gps global position
geometry_msgs/PoseStamped pose          # target pose   
geometry_msgs/TwistStamped velocity     # target velocity